.github/workflows/compute_impacted_tests.yaml

```diff
@@ -31,6 +35,6 @@ jobs:
         shell: bash
         run: |
           target_count=`cat ${{ steps.compute.outputs.impacted_targets_out }} | wc -l`
-          if [[ $target_count -ne 2 ]]; then
+          if [[ $target_count -ne 0 ]]; then
```

Comment:
Why this change?

Reply:
We moved sha computation outside of the compute_impacted_targets script, and expect it as an input into the script. As opposed to running tests with a stable test branch and a moving main, we now run tests with a stable test branch and a stable main, such that we can pin the shas. In that case, we would expect no impacted targets to be computed.